app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job = Job.find(params[:job_id])
+
+    @comment.save
```

Comment:
what happens when a comment doesn't save? how will a user know that their data didn't get saved correctly? 

i suggest doing a conditional here :thumbsup: